perun-base/src/main/resources/perun-roles.yml

```diff
@@ -8220,6 +8244,25 @@ perun_roles_management:
     assignable_to_attributes: false
     display_name: "Service user"
 
+  SPREGAPPLICATION:
+    primary_object:
+    assign_to_objects: {}
+    assignment_check:
+      - MFA:
+    entities_to_manage:
+      User: user_id
+    privileged_roles_to_manage:
+      - PERUNADMIN:
+      - VOADMIN: Vo
```

Comment:
I don't think we want to let VO admins to set / read this role.

Reply:
Also please add `skip_mfa: true` as we want to avoid enforcing MFA for the role.